.eslintrc.json

```diff
@@ -1,31 +1,25 @@
 {
     "root": true,
-    "extends": [
-        "eslint:recommended",
-        "plugin:prettier/recommended",
-        "plugin:@typescript-eslint/recommended"
-    ],
-    "plugins": [
-        "prettier",
-        "@typescript-eslint"
-    ],
+    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended","prettier"],
+    "plugins": ["@typescript-eslint","prettier"],
     "rules": {
         "eqeqeq": "error",
         "no-console": "warn",
-        "no-undef": "off",
-        "no-unused-vars": "off",
         "indent": ["error", 4],
         "camelcase": "error",
         "prettier/prettier": "error",
-        "@typescript-eslint/explicit-module-boundary-types": "off",
-        "@typescript-eslint/explicit-function-return-type": "off",
-        "@typescript-eslint/no-explicit-any": "error",
-        "@typescript-eslint/no-unused-vars": "warn"
+        "react/display-name": "off",
```

Comment:
why react rules in the SDK?

Reply:
i copied the other files settings to make sure everything same forgot to remove this line

Reply:
I think you need to remove all of these react rules

Reply:
removed